uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -110,6 +111,8 @@ define([
      * @param  {Array} leaveRequests - leave requests array from API
      */
     vm._indexLeaveRequests = function (leaveRequests) {
+      vm.leaveRequests = {};
```

Comment:
1. Fixed
2. It is necessary because when the calendar reloads because of a leave request is created/edited `vm.leaveRequests` should clear before reassigning values, otherwise duplicate values are shown in the UI.